# 11/27

# 1

- DNA
	- Polymer: nucleic acid
	- Monomer: nucleotides
	- More stable than RNA
		- RNA can self cleave
		- DNA should last as long as the cell, RNA should live as long as the "command"
	- Anti-parallel
		- One strand is 3' &rarr; 5' and the other is 5' &rarr; 3'
	- The DNA is not a nice symetrical shape
		- Major and minor groves
	- Absorbs UV light
		- The carbon rings in the bases of the backbone have alternating double and single bonds
	- Has a negative charge
		- Phosphate groups in the backbone
- Nitrogenous bases (pair):
	- Adenine (T)
	- Cytosine (G)
	- Guanine (C)
	- Thymine (A)
	- Uracil (only RNA)
		- Substatute for T
- Backbone:
	- Ribophosphate
	- 2 rings: purines (A, G)
	- 1 ring: pyrimidines (C, T, U)
	- Each pair is a 2 ring + 1 ring
		- Always 3 wide
		- TOP: Hydrogen bonds, SIDE: rings
		- 	|  | 2 | 3 |
			| :-: | :-: | :-: |
			| 1 | T | C |
			| 2 | A | G |
		- CG takes more energy to break
	- Connected by phosphodiester bonds
		- R-O-P(=O)-O-R
- Nucleosides:
	- Adenosine
	- Cytidine
	- Guanidine
	- Thymidine
	- Uradine
	- Naming:
		- ATP/CTP/SMP/TDP
		- Nucleoside + number of P + (P = phosphate)
			- A**T**P: T = Tri = 3
- Ribose
	- 5 carbon sugar
	- Dioxide: the H is on the 2nd carbon
		- Seperates DNA from RNA
	- 3' carbon: hydroxyl group
		- Attachs to next monomer in the chain
			- Elongates the chain
	- 5' carbon: phosphate
		- Always on the opposite end of the 3'

# 2

- 1 Chromosome: 1 double helix of DNA
	- Really long
	- Wrapped around 8 (octet) histones
		- Proteins
	- Then those DNA wrapped histones get wrapped
	- Have a "banding" pattern (look stripped)
		- Middle: centromere
			- Offcenter
				- P arm: shorter side
				- Q arm: longer side
		- Ending: telomere
			- Non-coding
	- Locus: a spot on a chromosome
	- Everything but bateria have the DNA wrapped around histones
- 3 billion base pairs total
	- 20,000 different genes that code for proteins
- 23 pairs (46 total)
	- Autosomes
		- 1 - 22
		- Hamologue: 1\` - 22`
			- Same size
			- Same genes (same locus will be in same gene)
			- Can different sequence (alleles)
				- Only 1/1000 of the nucleotides will be different
		- Biggest to smallest (1 = biggest)
	- Sex x-somes
		- XX
		- XY

# Cell Cycle

- Cell cycle
	- &rarr; M &rarr; G1 &rarr; S &rarr; G2 &rarr;
	- M: Mitosis
	- G: Gap ("growth")
		- G0: same as G1 but doesn't continue to S (end of cycle)
			- Example: neuron and muscle cells
	- S: Synthesis (replication)
- Cyclins
	- Regulate the cycle
	- Signal to the cell what part of the cycle they are in
	- Also multiple "checkpoints" between the S and M
		- Makes sure the cell is ready for the step
	- Managed by CDC (Cell Division Cycle)
		- Cyclin Dependent Kinase
			- Kinase: enzyme that adds a phosphate group
		- Oncogenes
			- Cause more activity of the cell cycle
		- Tumor Suppressor Genes (TS)
			- Cause less activity of the cell cycle
- Cancer cell
	- Either (or both):
		- Too much oncogenes: foot on gas
		- Not enough TS: no brake

## S phase

- Prokaryote: shape of DNA = circle
	- No histones
- Replication starts at the `origin of replication` (ori)
	- A specific sequence: consencus sequence
		- Other protein in the cell are looking for the this sequence
- Replication is bi-directional
	- Symetrical, what is going on on one half of is going on to the other in the other direction
- Helicase
	- Unwinds and splits the DNA
	- Works in both directions from the bubble
- DNA gyrase = Tepoisomerase
	- As helicase is unwinding the DNA, it cuts the DNA and then reconnects it
		- Releases the pressure caused by helicase unwinding
- Single stranded binding protein
	- Keeps the 2 strands split
	- Would want to rejoin immediately after the helicase
- DNA polymerase
	- Build 5\` to 3`
		- 5 made first
		- Adds on to the 3` end
	- Adds dATP/dGTP/dCTP/dTTP
		- d: deoxyribose
		- Takes one of them in, adds the base and releases 2 phosphates
	- Exonuclege
		- DNA polymerase makes very little mistakes, but this step is a proofreading step
		- Undoes a mistake
	- DNA polymerase III
		- The main one building
		- 5\` to 3\` building
		- 3\` to 5\` proofreading
	- DNA polymerase I
		- Same 5\`&rarr;3\` building and 3\`&rarr;5\` proofreading
		- But also 5\` to 3` econuclease
			- Removes the RNA primer
	- The new strand is "reading" off the old strand and is inserting the complementary base
		- The new strand is complementary and anti-parrallel to the read strand
	- DNA polymerase can not "just start"
		- Can build off of something, but needs help starting
		- Needs a hydroxyl group to start with
		- Primase first puts down a RNA primer for the DNA polymerase to get started
- Leading strand = continuous strand
	- The polyermase is going in the same direction as the helicase
- Lagging strand = discontinous strand
	- The polyermase is going in the opposite direction as the helicase
	- The polyermase is going to build the strand in small pieces
		- Okazaki fragments
		- But still needs the primase to get started 
	- DNA ligase
		- Glues the Okazaki fragments together
		- Glues the 3\` end of one fragment to the 5` end of the next
		- Diphophodiester bond
- Parent: original DNA
- Daughter: new DNA
	- Semi-conservative
		- 1 strand is completely from the parent
		- 1 strand is completely new
			- Half lagging, half leading

# Transcription

- DNA &rarr; RNA
	- Still nucleic acid
- Gene:
	- Unit of transcription
- Transcription starts at a `promoter`
	- A concenus sequence
	- TATA box: part of the promoter/concenus sequence
- RNA polymerase builds the RNA
	- TSS: Transcription Start Site (= +1 site)
		- Where the RNA polymerase starts
	- Builds 5' to 3' anti-parrellel
		- The built strand is the same as the "sense" strand
		- Uses the into (reads) from the "antisense" strand
	- Ends at the terminator
	- Does
- rRNA
	- Ribosomal RNA
	- Makes up the ribosome
	- Made in the nucleolus
- mRNA
	- Messenger RNA
	- Carries the message from the DNA to the ribosome
- RNases
	- "Eat" RNA
	- 5' cap
		- Protects the mRNA from being degraded
		- Has a 'G' (methal guanine) that the RNase doesn't recognize so it stops
	- Poly-A tail
		- Protects the mRNA from being degraded
		- A stack of As as a buffer that the Rnase slowly eats through
	
# Translation

- RNA &rarr; protein
	- Nucleic acids &rarr; protein
	- Happens in the cytoplasm
- Ribosome binds on the 5' end of the RNA
	- Scans until the AUG (start codon)
		- Full ribosome assembles
			- 3 spots: E, P, A
	- Ends at a stop codon
	- Builds N terminus to the C terminus
- The AUG starts in the A spot
- tRNA comes into A
	- A: accepts the anti-codon matching the codon in the RNA
	- Anti-code for AUG: UAC
	- Uses this code to pick the amino acid to use
- Takes a "3 letter" step
	- The AUG is now in the P spot
	- P stacks the chosen amino acid in A on to the next chosen amino acid
- E: exit
	- tRNA leaves